src/components/monitoringTable/MonitoringTable.tsx

```diff
@@ -0,0 +1,185 @@
+      type: 'autocomplete-multiple',
+      label: t('serviceStateFilter'),
+      options: [
+        { value: 'ONLINE', label: 'online' },
+        { value: 'OFFLINE', label: 'offline' },
+        { value: 'N_D', label: 'n/d' },
+      ],
+    },
+  ])
+
+  const { paginationParams, paginationProps, getTotalPageCount } = usePagination({
+    limit: import.meta.env.VITE_PAGINATION_LIMIT
+      ? parseInt(import.meta.env.VITE_PAGINATION_LIMIT)
+      : 10,
+  })
+
+  async function fetchServices(
+    paginationParams: any,
+    filtersParams: any
+  ): Promise<GetServicesResponseType> {
+    let payload = {
+      offset: paginationParams.offset,
+      limit: paginationParams.limit,
+      producerName: filtersParams.producerName,
+      versionNumber: filtersParams.versionNumber,
+      eserviceName: filtersParams.eserviceName,
+      state: filtersParams.state,
+    }
+    let response = apiRequests.getServicesApi(payload as getServicesType)
+    return response
+  }
+
+  const { data: services, isInitialLoading } = useQuery({
+    queryKey: ['services', paginationParams, filtersParams],
+    queryFn: () => fetchServices(paginationParams, filtersParams),
+    onError: (error) => updateSnackbar(true, 'Errore nella richiesta', 'error'),
+  })
+
+  const goDetails = (serviceId: string) =>
+    navigate(
+      resolvePathVariables('/monitoring/serviceDetails/:serviceId', {
+        serviceId: serviceId,
+      })
+    )
+
+  return (
+    <>
+      <Filters {...handlers} />
+      {isInitialLoading ? (
+        <TableSkeleton headLabels={headLabels} />
+      ) : (
+        <Table
+          isEmpty={services?.content ? services.content.length === 0 : true}
+          headLabels={headLabels}
+          noDataLabel={t('noResultsTable')}
+        >
+          {map(services?.content, (service) => (
+            <TableRow
+              key={service.id}
+              cellData={[
+                service.eserviceName as string,
+                service.versionNumber as string,
+                service.producerName as string,
+                <Chip
+                  key={service.id}
+                  label={toLower(service.state)}
+                  color={
+                    service.state === 'ONLINE'
+                      ? 'success'
+                      : service.state === 'OFFLINE'
+                      ? 'error'
+                      : 'warning'
+                  }
+                />,
+                service.responseReceived
+                  ? format(new Date(service.responseReceived), 'dd-MM-yyyy') +
+                    ', ore ' +
+                    format(new Date(service.responseReceived), 'HH:mm')
+                  : '',
+              ]}
+            >
+              <ButtonNaked
+                size="small"
+                color="primary"
+                component="button"
+                onClick={() => goDetails(toString(service.id))}
+              >
+                {t('readMore')}
+              </ButtonNaked>
```

Comment:
Questo componente nel DOM viene renderizzato con un tag `button` quando in realtà, per questioni di accessibilità, dovrebbe essere un anchor tag (`a`). 
In generale, in questi casi si consiglia la seguente implementazione:

```suggestion
              <ButtonNaked
                size="small"
                color="primary"
                component={Link}
                to={`/monitoring/serviceDetails/${toString(service.id)}`} // o con generatePath, a vostra discrezione
              >
                {t('readMore')}
              </ButtonNaked>
```

Da rivedere.